docs/get-started/getting-started-gcp.md

```diff
@@ -0,0 +1,113 @@
+
+
+ 
+## How to spin up a Dataproc Cluster Accelerated by GPU
+ 
+ You can use [Cloud Shell](https://cloud.google.com/shell) to execute shell commands that will create a Dataproc cluster. Cloud Shell contains command line tools for interacting with Google Cloud Platform, including gcloud and gsutil. Alternatively, you can install [GCloud SDK](https://cloud.google.com/sdk/install) on your laptop. From the Cloud Shell, users will need to enable services within your project. Enable the Compute and Dataproc APIs in order to access Dataproc, and enable the Storage API as you’ll need a Google Cloud Storage bucket to house your data. This may take several minutes.
+```bash
+gcloud services enable compute.googleapis.com
+gcloud services enable dataproc.googleapis.com
+gcloud services enable storage-api.googleapis.com
+``` 
+
+After command line environment is setup, log in to your GCP account. We can now create a Dataproc cluster with configuration mentioned below.
+The configuration will allow users to run any of the [notebooks demo](../demo/GCP) on GCP. Alternatively, users can also start a 2*2T4 worker nodes.
+* [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) and [RAPIDS Acclerator for Apache Spark](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids) through initialization actions
+* One 8-core master node and 5 32-core worker nodes
+* Four NVIDIA T4 to each worker nodes
+* [Local SSDs](https://cloud.google.com/dataproc/docs/concepts/compute/dataproc-local-ssds) is recommended to improve IO for Spark scratch places  
+* Component gateway enabled for accessing Web UIs hosted on the cluster
+* Configuration for [GPU scheduling and isolation](/get-started/yarn-gpu.html)
+
+
+```bash
+    export REGION=[Your Prefer GCP Region]
+    export GCS_BUCKET=[Your GCS Bucket]
+    export CLUSTER_NAME=[Your Cluster Name]
+    export NUM_GPUS=4
+    export NUM_WORKERS=5
+
+gcloud dataproc clusters create $CLUSTER_NAME  \
+    --region $REGION \
+    --image-version=preview-ubuntu \
+    --master-machine-type n1-standard-16 \
+    --num-workers $NUM_WORKERS \
+    --worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS \
+    --worker-machine-type n1-highmem-32\
+    --num-worker-local-ssds 4 \
+    --initialization-actions gs://dataproc-initialization-actions/gpu/install_gpu_driver.sh,gs://dataproc-initialization-actions/rapids/rapids.sh \
+    --optional-components=ANACONDA,JUPYTER,ZEPPELIN \
+    --metadata gpu-driver-provider="NVIDIA" \
+    --metadata rapids-runtime=SPARK \
+    --bucket $GCS_BUCKET \
+    --enable-component-gateway \
+    --properties="^#^spark:spark.yarn.unmanagedAM.enabled=false#spark:spark.task.resource.gpu.amount=0.125#spark:spark.executor.cores=8#spark:spark.task.cpus=1#spark:spark.executor.memory=8G"`
```

Comment:
Just like for the databricks version can we have an explanation for each of these configs? Also why are the configs different here?  Should we not have a standard for how we expect the configs to be setup?

Reply:
`spark:spark.yarn.unmanagedAM.enabled=true` moves yarn application master to spark driver, so that we fully utilize the cluster, I will remove it once it stops break the sparkUI in the dataproc v2 preview image. As for the rest, I included them in the init action and notebooks, I have asked Karthik to remove those.

Reply:
I will update this.